realtime.js

```diff
@@ -145,3 +145,20 @@ exports.waittimes = function(times) {
         times: times
     });
 }
+
+// Takes in a list of tas to notify about their help time
+exports.notifytime = function(notif_tas) {
+    exports.seq = exports.seq + 1;
+    if (!sio) {
+        console.log("ERROR: Socket.io is not initialized yet");
+        return;
+    }
+
+    notif_tas.forEach(ta => {
+        sio.emit("notifytime", {
```

Comment:
This should be `sio.to(ta_room).emit` since we don't need to notify students about TA reminders. Now that we're removing the increment of `seq`, it shouldn't cause anything to be out of sync.